src/operator/nn/mkldnn/mkldnn_act.cc

```diff
@@ -134,7 +134,11 @@ class MKLDNNActForward {
 static MKLDNNActForward &GetActForward(const ActivationParam& param,
                                        const OpContext &ctx, const NDArray &in_data,
                                        const mkldnn::memory &in_mem) {
+#if DMLC_CXX11_THREAD_LOCAL
   static thread_local std::unordered_map<MKLDNNActSignature, MKLDNNActForward, OpHash> fwds;
+#else
+  static MX_THREAD_LOCAL std::unordered_map<MKLDNNActSignature, MKLDNNActForward, OpHash> fwds;
```

Comment:
do we need to make this change here too: https://github.com/apache/incubator-mxnet/blob/35fb9ea1093bd0006a396b5e493e5949b66ca33e/src/c_api/c_api_profile.cc#L115 ?

Reply:
i think so.